cli/src/pcluster/cli/commands/dcv_connect.py

```diff
@@ -64,10 +64,11 @@ def _dcv_connect(args):
 
         try:
             url = _retry(_retrieve_dcv_session_url, func_args=[cmd, args.cluster_name, head_node_ip], attempts=4)
-            url_message = f"Please use the following one-time URL in your browser within 30 seconds:\n{url}"
+            url_message = f"Please use the following one-time URL in your browser within 30 seconds:"
```

Comment:
There is a test that should be changed accordingly: https://github.com/aws/aws-parallelcluster/blob/ea57725c46392c175aa66a2c8a8fcc9e8e7ec2a2/tests/integration-tests/tests/dcv/test_dcv.py#L76-L79